src/styles/header.scss

```diff
@@ -0,0 +1,21 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $header-bg;
+  width: calc(100% - #{$header-padding * 2});
```

Comment:
The variable `$header-padding` is used here, but it is not defined within this file. Ensure that `$header-padding` is defined in a file that is imported into this one, or define it here if it's missing.